cmd/terramate/e2etests/exp_eval_test.go

```diff
@@ -223,4 +223,117 @@ func TestExpEval(t *testing.T) {
 	}
 }
 
+func TestGetConfigValue(t *testing.T) {
+	type (
+		globalsBlock struct {
+			path string
+			add  *hclwrite.Block
+		}
+		testcase struct {
+			name    string
+			layout  []string
+			wd      string
+			globals []globalsBlock
+			expr    string
+			want    runExpected
+		}
+	)
+
+	testcases := []testcase{
+		{
+			name: "boolean expression",
+			expr: `true || false`,
+			want: runExpected{
+				Status:      1,
+				StderrRegex: "expected a variable accessor",
+			},
+		},
+		{
+			name: "funcall expression",
+			expr: `tm_upper("a")`,
+			want: runExpected{
+				Status:      1,
+				StderrRegex: "expected a variable accessor",
+			},
+		},
+		{
+			name: "unsupported rootname",
+			expr: `local.value`,
+			want: runExpected{
+				Status:      1,
+				StderrRegex: "only terramate and global variables are supported",
+			},
+		},
```

Comment:
for completeness maybe test when accessing an undefined metadata/global.

Reply:
added